miner/miner.go

```diff
@@ -187,6 +193,8 @@ func (miner *Miner) SetMaxDASize(maxTxSize, maxBlockSize *big.Int) {
 		miner.config.MaxDABlockSize = new(big.Int).Set(maxBlockSize)
 	}
 	miner.confMu.Unlock()
+	maxDABlockSizeGuage.Update(miner.config.MaxDABlockSize.Int64())
+	maxDATxSizeGuage.Update(miner.config.MaxDATxSize.Int64())
```

Comment:
Do you think we ought to take more care to get a read lock when reaching into the `miner.config`? I notice that we aren't doing this in `miner/worker.go` (the other place this config gets read). 

Another option here would be to use the `maxTxSize, maxBlockSize` input variables rather than reaching into the miner config.

Reply:
Yeah I noticed that. Even though its an RWMutex it only seems to be used as a write lock so its probably fine from what I understand 

I will defer to you if you think this should be inside the lock though as im much less familiar with the codebase and unsure if this can have any weird side effects